### ISSUE TITLE
Enhancement: Better reverse driving skills

### ISSUE THREAD

**Issue by blubstar86** [Stephan-S/FS22_AutoDrive#1122]
Hello, i wish AD would not lock front axel of trailers to reverse them.

In my opinion it should be possible to reverse trailers without doing it.

The mod "Trailer Assist" from mogli gives maybe some hints to do it.

[https://github.com/Mogli12/TrailerAssist](url)

@Tensuko, perhaps CP can benefit from it too. (I had some jack knifing trailers when they reverse in combine unload mode)

https://github.com/Stephan-S/FS22_AutoDrive/assets/47928255/0fe19a23-0e95-4c50-b5be-0afbfcae6d51

**Comment by Tensuko** [comment]
Nope. We have a real reverse code based on math. Sure some angles cant work but thats how it is and we will keep that. Nothing will make me change my mind about that.
